sys/linux/fs_ioctl.txt

```diff
@@ -13,8 +13,9 @@ ioctl$FICLONE(fd fd, cmd const[FICLONE], arg fd)
 ioctl$FICLONERANGE(fd fd, cmd const[FICLONERANGE], arg ptr[in, file_clone_range])
 ioctl$FIDEDUPERANGE(fd fd, cmd const[FIDEDUPERANGE], arg ptr[in, file_dedupe_range])
 
-ioctl$FS_IOC_GETFLAGS(fd fd, cmd const[FS_IOC_GETFLAGS], arg ptr[out, intptr])
-ioctl$FS_IOC_SETFLAGS(fd fd, cmd const[FS_IOC_SETFLAGS], arg ptr[in, intptr])
+ioctl$FS_IOC_GETFLAGS(fd fd, cmd const[FS_IOC_GETFLAGS], arg ptr[out, flags[fs_flags, intptr]])
```

Comment:
These ioctls actually take a pointer to an `int32`, not a pointer to an `intptr`.  Can you fix that too?  Otherwise this change looks good.

Reply:
What about?
```
ioctl$EXT4_IOC_SETFLAGS(fd fd, cmd const[EXT4_IOC_SETFLAGS], arg ptr[in, flags[ext4_inode_flags, intptr]])
```

Should I change this one too?

Reply:
It's actually the same ioctl, so it should just be removed from `fs_ioctl_ext4.txt`.  Along with `ext4_inode_flags`, which likewise is the same as `fs_flags`.